packages/connection-form/package.json

```diff
@@ -38,9 +38,9 @@
     "depcheck": "depcheck",
     "check": "npm run lint && npm run depcheck",
     "check-ci": "npm run check",
-    "test": "mocha",
-    "test-electron": "xvfb-maybe electron-mocha --no-sandbox",
-    "test-cov": "nyc -x \"**/*.spec.*\" --reporter=lcov --reporter=text --reporter=html npm run test",
+    "test": "cross-env COMPASS_CSFLE_SUPPORT='true' mocha",
```

Comment:
The way we normally work with FF locally is this:

```
export COMPASS_CSFLE_SUPPORT='true'

npm run any-kind-of-test
```

This is intentional, we want the CI to always test what's released not what will be released when the FF is enabled.